lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset.py

```diff
@@ -834,6 +853,13 @@ def vm_size_is_valid(self):
                 return True
         return False
 
+    def get_network_security_group(self, name):
```

Comment:
can we get the security group from other resource group? In ansible we can handle this by refering it with resource id and adding a `network_security_group_resource_group` field.

Reply:
Addressed this with latest changes. Please review.